index.html

```diff
@@ -14,172 +14,200 @@
+                    </div>
+                    <div class="work-lang">
+                        <ul>
+                            <li>
+                                <p class="lang-text">html</p>
+                            </li>
+                            <li>
+                                <p class="lang-text">css</p>
+                            </li>
+                            <li>
+                                <p class="lang-text">javascript</p>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="button" class="work-btn btn">
+                        See Project
+                    </button>
                 </div>
-                <button type="button" class="work-btn btn">
-                    See Project
-                </button>
             </div>
         </section>
         <section class="about-me">
-            <h1 class="header-text">About
-                Myself</h1>
-            <p>
-                I’m a software developer! I can help you build a product , feature or website Look through some of my
-                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-                contact me.
-            </p>
-            <a href="#">LET’S CONNECT</a>
-            <ul class="social-links-about">
-                <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
-                <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
-                <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
-                <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
-                <li> <a href="#"><img src="img/icons/Vector3.png" alt=""></a></li>
-            </ul>
-            <button type="button" class="about-btn btn">
+            <div class="about-info">
+                <h1 class="header-text">About
+                    Myself</h1>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <ul class="social-links-about">
+                    <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
+                    <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
+                    <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
+                    <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
+                    <li> <a href="#"><img src="img/icons/Vector3.png" alt=""></a></li>
+                </ul>
```

Comment:
- [ ] there is a gap between your social media icon and it would be better if you use remove it in order to match the Design 
- Figma
![image](https://user-images.githubusercontent.com/61790587/171681508-7a1e05ec-a6ac-4cf8-90f3-9dc3373ec2ee.png)
- Yours
![image](https://user-images.githubusercontent.com/61790587/171681525-1c0baede-b118-463e-983e-6320a3231498.png)